frontend/src/formdescriptioneditor/PieChartWidget.tsx

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { PieChart } from 'charts/Charts.types';
+import { PieChartComponent } from 'charts/pieChart/PieChartComponent';
+import { WidgetProps } from 'formdescriptioneditor/WidgetEntry.types';
+import React, { useEffect, useRef, useState } from 'react';
+
+const useStyles = makeStyles((theme) => ({
+  selected: {
+    color: theme.palette.primary.main,
+  },
+}));
+
+export const PieChartWidget = ({ widget, selection }: WidgetProps) => {
+  const classes = useStyles();
+  const chart = {
```

Comment:
Declare your variable `chart` like `const chart: PieChart = {...}` instead of `const chart = {...} as PieChart`

Reply:
ok